kythe/release/appengine/buildbot/cache-llvm.sh

```diff
@@ -32,6 +32,15 @@
 
 CACHE="$HOME/.cache/kythe-llvm"
 
+copy() {
+  if which gcp &>/dev/null; then
```

Comment:
I assume this is because macOS `cp` doesn't support `-l`?

I think you're going to have to add `coreutils` to the installation list, because it isn't by default even if Homebrew is present.